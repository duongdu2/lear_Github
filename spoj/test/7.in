30
6
8573 1370 4463 5182 6837 9731 
30
5749 3688 1795 3193 9770 3779 1481 6123 7252 4548 272 4591 2070 2306 2071 1898 5629 9439 9012 9452 6789 9608 2555 2570 5148 1011 8148 9378 6848 7161 
33
8664 9910 7328 2438 5479 7154 9321 4771 6501 5544 2197 4695 9078 9091 3231 9780 6014 9927 6464 5165 7103 8521 1927 8752 9998 8112 4766 9235 8515 3409 6653 1361 4815 
3
3896 2332 8341 
49
4925 4600 1824 2005 400 1591 5067 2431 4802 8355 5412 7721 7628 592 1825 7114 6528 7703 6369 7904 516 3866 7120 4743 5919 1393 6266 4006 939 1048 4086 6758 8194 9014 3857 3462 5071 1127 4119 8463 8728 100 4494 6672 4409 6169 9705 2614 3649 
27
4850 7148 46 1103 3854 2139 495 4702 5879 5589 730 4553 5197 4135 1891 6914 9631 5851 8087 706 7838 4106 193 8580 8258 7114 4982 
26
1018 68 1061 5649 2532 5120 7945 1155 820 6230 9342 7559 8160 1447 3921 9813 1022 8789 2334 7849 1745 9735 6416 711 1178 3039 
27
7624 9588 5080 9141 8409 7832 6041 4121 5195 4098 3743 9403 3738 67 906 6386 6708 8819 1808 9228 8755 9293 5231 269 9054 8209 1344 
47
8938 4854 9125 6812 1593 9574 400 5883 9697 1001 428 3391 5555 1864 5768 8078 3654 3173 3176 7391 5501 5500 3694 7035 4557 341 576 6959 2228 1256 2005 4141 8349 8057 5689 2795 7086 9038 517 6684 3972 852 4921 5805 8313 9242 7759 
16
3055 2019 4589 4560 3241 8941 6031 5729 4448 8307 4626 8530 7185 3565 2128 6678 
30
6669 8865 5170 9800 8760 559 5319 7966 7037 5378 7801 3744 6892 2260 1917 9049 650 9830 5259 1189 5202 5681 3680 8093 7094 3826 7495 1714 2997 8792 
9
8590 5155 6816 288 3890 7055 5771 4231 6158 
10
7675 275 160 6110 7848 11 6085 3682 990 5453 
44
864 4611 1684 4593 3445 422 9394 4339 3164 6544 6089 4050 5341 7482 7106 735 9962 8305 7069 5620 7556 9603 5850 3376 1510 9026 8962 3911 4417 7362 9221 9692 3663 8625 682 6314 8443 4882 4636 796 3390 964 2090 6935 
5
7541 663 5586 4101 8707 
9
5879 2265 8522 385 7440 4124 8987 9784 5135 
29
4182 8122 6377 5830 9928 3357 4209 1557 1211 9071 5519 8592 4104 7818 6679 6800 103 6564 4512 2367 9649 6161 551 8529 415 5683 6545 8576 3526 
15
5570 5394 3840 1229 5440 9383 1987 4619 6171 3038 2190 9336 4541 4230 6625 
43
3232 5628 6690 5325 2678 6086 1792 101 4438 5617 3015 8458 1516 3212 6604 7031 1758 1463 7920 7963 7016 4640 2531 1850 2013 3985 2491 6832 1175 8036 974 4421 5009 2278 4880 6463 8836 5606 6095 7378 1745 1415 4982 
25
4703 4070 9313 2192 5193 5945 717 2899 308 8252 8483 4994 4674 9057 1017 4467 1418 1464 1003 7373 2821 3977 7828 4882 6145 
45
9190 6819 517 2778 2372 4042 5466 9062 4931 5587 7869 6286 912 3427 668 687 7044 7567 4258 5182 9441 8780 3112 2757 4390 8196 9344 7771 9734 938 4047 7411 7706 1529 6741 8872 4137 9008 5771 4732 6705 4135 7593 6364 8860 
13
9366 1444 6470 947 803 1994 4007 9000 7714 8809 9816 8015 9142 
33
5059 9747 7496 2478 6842 5211 2705 2290 6371 4188 2099 7183 4914 4511 4196 7218 9955 392 987 3386 4263 7707 6064 9532 9319 6583 9376 923 5654 2989 519 462 8960 
32
9008 5634 2758 6876 8625 2817 8435 3616 1705 4942 2207 5195 7753 4410 8920 4337 4305 9782 6541 5612 2180 5071 1961 9483 3278 7715 1405 9086 9424 286 203 3271 
36
577 5888 3854 2003 5812 9179 4653 2493 9053 7357 3388 138 3536 5140 601 1692 6953 1932 9823 1976 3345 9632 8598 4807 5440 8396 9443 8837 7403 5422 5962 565 8487 8097 190 8645 
3
710 7258 1412 
32
6107 6999 5635 6935 5373 7829 4840 8600 7527 1232 259 1629 9365 9806 4972 3663 6872 5761 3318 4253 2531 1520 2062 8286 7689 9287 7475 8255 2980 3378 431 54 
37
5211 1284 3309 327 1585 7574 6262 5184 5019 4654 3338 881 738 4187 7552 5267 61 9002 5401 5197 5648 8320 1637 8894 267 3162 8451 4644 6065 4122 2022 8339 5986 8624 3857 9299 7864 
18
4862 1506 1518 7908 9743 5447 7785 5702 9800 8053 6983 8393 6297 9563 8823 1531 7220 5277 
35
6454 8695 2133 5199 5089 5817 480 1033 4802 2458 3893 6364 8060 3441 1016 3660 6491 2079 4391 7711 8696 9770 9827 9103 2699 3594 2342 5160 5136 8371 950 1778 5095 3985 7810 
