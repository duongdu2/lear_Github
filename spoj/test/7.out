25141
28830
29835
14569
27447
26469
28890
28284
28513
25778
28679
22461
21633
29257
21834
27293
28648
25344
25330
26853
28519
28324
29234
28689
28898
9380
28458
27195
29106
28700
